### PR TITLE
Return asset data for type 6 transactions - Closes #2194

### LINKED ISSUE #2194: No asset data is returned for transactions type 6 (inTransfer)
### Expected behavior
I expected to see `asset` object data when requesting `/api/transactions?type=6`

### Actual behavior
Empty object is returned

### Steps to reproduce
`curl testnet.lisk.io:7000/api/transactions?type=6`

### Which version(s) does this affect? (Environment, OS, etc...)
Testnet

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?
No asset data was returned when querying type 6 transactions (in-transfer).

### How did I fix it?
The SQL query was not case-sensitive, added quotes around `in_dappId` to make the query successful.
Without quotes, it would always search for lower case.

### How to test it?
`npm run mocha -- test/unit/db/repos/transactions/index.js`

### Review checklist

* The PR solves #2194 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/fixtures/accounts.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ accounts.existingDelegate = {
 	publicKey: 'addb0e15a44b0fdc6ff291be28d8c98f5551d0cd9218d749e30ddb87c6e31ca9',
 	passphrase:
 		'actress route auction pudding shiver crater forum liquid blouse imitate seven front',
-	balance: '0',
+	balance: '1000000000',
```
Comment (diego-G) [inline test/fixtures/accounts.js]: Is this intended?

Comment (Tschakki) [reply test/fixtures/accounts.js]: this doesn't seem necessary, removed it!

File: test/functional/http/get/transactions.js [hunk 2]
```diff
@@ -865,5 +913,46 @@ describe('GET /api/transactions', () => {
 				});
 			});
 		});
+		/* eslint-disable mocha/no-skipped-tests */
```
Comment (MaciejBaj) [inline test/functional/http/get/transactions.js]: Can we add a comment clarifying why this exact test is skipped and when will be unskipped?

Comment (Tschakki) [reply test/functional/http/get/transactions.js]: good idea!
I added the comment now

**Pull request merged**